tecken/librequests.py

```diff
@@ -3,8 +3,7 @@
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
 import requests
-from requests.adapters import HTTPAdapter
-from requests.packages.urllib3.util.retry import Retry
+from requests.adapters import HTTPAdapter, Retry
```

Comment:
This file was included in this PR completely by accident. It should have been on a different branch where I collected some random clean-ups. If you don't mind, I'll just keep it here now that you already reviewed the changes.

The reason for this particular change is that the VS Code Python language server was unable to deal with [this rather special code](https://github.com/psf/requests/blob/769bc3ae5cc8cb42242e4505b310dd10889cb54c/src/requests/packages.py#L15-L21), which illegally modifies the dictionary returned by `locals()` and monkey-patches modules into `sys.modules` at some legacy path.

Reply:
Ewww...

This is a thing I copy/paste from project to project because requests doesn't have a default timeout and we often need to retry responses for certain HTTP status codes and I never got around to settling on a single retryable library. (There's [bug 1774667](https://bugzilla.mozilla.org/show_bug.cgi?id=1774667) which covers switching Socorro to tenacity.) We should fix Socorro and crashstats-tools, too.